### PR TITLE
Fix/mandatory date

### PR DESCRIPTION
**Pull request by mAkeddar**

### CODE REVIEW

File: src/Components/PageComponents/SingleTripContent.tsx [hunk 1]
```diff
@@ -35,23 +35,28 @@ function SingleTripContent(activeTab: any) {
   }
 
   useEffect(() => {
-    console.log(day);
+    console.log('ALORS', day);
```
Comment (hugo2410) [inline src/Components/PageComponents/SingleTripContent.tsx:38]: remove the alors

File: src/Components/PageComponents/SingleTripContent.tsx [hunk 2]
```diff
@@ -35,23 +35,28 @@ function SingleTripContent(activeTab: any) {
...
     if (day != null) {
       setDaySelected(true);
     } else setDaySelected(false);
   }, [day]);
 
   useEffect(() => {
-    console.log(dayTime.length);
+    console.log(dayTime);
```
Comment (hugo2410) [inline src/Components/PageComponents/SingleTripContent.tsx:45]: I would actually remove all the console logs

File: src/Components/PageComponents/SingleTripContent.tsx [hunk 3]
```diff
@@ -35,23 +35,28 @@ function SingleTripContent(activeTab: any) {
...
     } else setDayTimeSelected(true);
   }, [dayTime]);
 
   useEffect(() => {
-    if (defaultDayTime.length === 0) {
+    console.log(routeInfo.time);
+    if (routeInfo.time.length === 0) {
       setDayTimeSelected(false);
     } else setDayTimeSelected(true);
+    console.log('ALed', routeInfo.timeRange);
```
Comment (hugo2410) [inline src/Components/PageComponents/SingleTripContent.tsx:56]: same

File: src/Components/PageComponents/SingleTripContent.tsx [hunk 4]
```diff
@@ -35,23 +35,28 @@ function SingleTripContent(activeTab: any) {
...
-    console.log(dayTime.length);
+    console.log(dayTime);
     if (dayTime && dayTime.length === 0) {
       setDayTimeSelected(false);
     } else setDayTimeSelected(true);
   }, [dayTime]);
 
   useEffect(() => {
-    if (defaultDayTime.length === 0) {
+    console.log(routeInfo.time);
```
Comment (hugo2410) [inline src/Components/PageComponents/SingleTripContent.tsx:52]: same

File: src/Pages/Profile/MyAccount.tsx [hunk 5]
```diff
@@ -21,6 +21,7 @@ const MyAccount: React.FC = () => {
 
   useEffect(() => {
     if (user) {
+      console.log(user);
```
Comment (hugo2410) [inline src/Pages/Profile/MyAccount.tsx:24]: remove

**Comment by hugo2410** [comment]
image
Transaltions are not set up properly for some of the fields + there is an issue with the initial value of the fields

File: src/Pages/Sender/EnterRequestSizeWeightImage.tsx [hunk 6]
```diff
@@ -107,6 +123,7 @@ const EnterRequestSizeWeightImage: React.FC = () => {
             onClick={() => {
               logEvent(analytics, 'send_2_sizeWeight_next_button_click');
             }}
+            disabled={!weightSelected || !sizeSelected}
```
Comment (hugo2410) [inline src/Pages/Sender/EnterRequestSizeWeightImage.tsx:126]: I believe we want both no? should not be and?

File: src/Pages/Sender/EnterRequestTime.tsx [hunk 7]
```diff
@@ -46,24 +46,30 @@ const EnterTime: React.FC = () => {
   }, [dayTime]);
 
   useEffect(() => {
-    if (dayRange == null) {
+    if (dayRange == null || dayRange.length === 0) {
+      console.log(dayRange);
```
Comment (hugo2410) [inline src/Pages/Sender/EnterRequestTime.tsx:50]: del

File: src/Pages/Sender/EnterRequestTime.tsx [hunk 8]
```diff
@@ -46,24 +46,30 @@ const EnterTime: React.FC = () => {
...
+      defaultDateRange = [
+        dayjs(objecInfo.dateRange[0], 'YYYY-MM-DD'),
+        dayjs(objecInfo.dateRange[1], 'YYYY-MM-DD'),
+      ];
+      setDayRangeSelected(true);
+    } else {
+      setDayRangeSelected(false);
+    }
+    console.log(defaultDateRange);
+  }, []);
```
Comment (hugo2410) [inline src/Pages/Sender/EnterRequestTime.tsx:73]:
```suggestion
   const isDateRangeValid = objecInfo.dateRange[0] !== '' && objecInfo.dateRange[1] !== '';
  setDayRangeSelected(isDateRangeValid);

  if (isDateRangeValid) {
    defaultDateRange = [
      dayjs(dateRange[0], 'YYYY-MM-DD'),
      dayjs(dateRange[1], 'YYYY-MM-DD'),
    ];
  }

```

**Comment by mAkeddar** [comment]
I pushed a fix for the comments + Removed all the newthings from myaccount (don't know why it got here from another branch)
Also fixed a bug in the weight size screen

**Comment by hugo2410** [comment]
Looks good! @mAkeddar

**Pull request merged**